changelog.d/20250731_113700_8730430+m1yag1_sc_43139_update_warn_deprecated.rst

```diff
@@ -0,0 +1,5 @@
+Development
+-----------
```

Comment:
```suggestion
Changed
-------
```

Nitpick, but this is a user-facing change as far as I'm concerned, since there's a new external symbol which users will care about in the future.